doc/classes/GPUParticles2D.xml

```diff
@@ -96,6 +96,7 @@
 		</member>
 		<member name="preprocess" type="float" setter="set_pre_process_time" getter="get_pre_process_time" default="0.0">
 			Particle system starts as if it had already run for this many seconds.
+			[b]Note:[/b] This can be very expensive if set to a high number as it requires running the particle shader a number of times equal to the [member fixed_fps] (or 30, if [member fixed_fps] is 0) for every second. In extreme cases it can even lead to a GPU crash due to the volume of work done in a single frame.
```

Comment:
Maybe...? For most people the in-depth explanation may be a bit overboard, but I can't say that with absolute confidence. I mean, I do like it, But it feels like it's talking more about `fixed_fps` than `preprocess`.
```suggestion
			[b]Warning:[/b] Setting this property to a very high value can significantly reduce performance and, in extreme cases, lead to a GPU crash. If absolutely necessary, this can be mitigated by lowering the [member fixed_fps], so long as [member fixed_fps] is not [code]0[/code].
```
Or similar.

I also believe this should be a warning as we're talking about a potential crash, or performance so poor it basically becomes impossible to use the editor (as I can gather from the original issue)

Reply:
This suggestion isn't correct. It's not a matter of low performance. It will cause a freeze and with extreme enough values on low-end enough systems you will just get a crash. 

> Maybe...? For most people the in-depth explanation may be a bit overboard, but I can't say that with absolute confidence. I mean, I do like it, But it feels like it's talking more about fixed_fps than preprocess.

I'm not sure what you mean. Your suggestion is way more focused on fixed_fps than mine...

> I also believe this should be a warning as we're talking about a potential crash, or performance so poor it basically becomes impossible to use the editor (as I can gather from the original issue)

The performance being so bad it stalls the editor part of the bug was already fixed. This PR is just about the GPU crash